src/nr_openai_observability/monitor.py

```diff
@@ -8,12 +8,11 @@
 
 
 def initialization(
-    application_name: str,
+    application_name: str = "",
```

Comment:
I realize that the `application_name` still being accessible here is to maintain backwards compatibility for existing callers. Would you mind adding a warning log if the value is set (is not None) to let callers know that this argument is no longer used? With the variable still being defined and accepted, it gives the impression that passing a value in will actually do something.